frontend/src/features/documents/list/DocumentListView.tsx

```diff
@@ -84,13 +113,36 @@ export const DocumentListView: React.FunctionComponent<IDocumentListViewProps> =
           sort={sort}
           setSort={setSort}
           onViewDetails={handleViewDetails}
-          onDelete={handleDelete}
+          onDelete={handleDeleteClicked}
         />
       </Styled.Scrollable>
       <DocumentDetailModal
         display={isDetailsVisible}
         setDisplay={setIsDetailsVisible}
         pimsDocument={selectedDocument}
+        handleClose={handleViewDetailsClose}
+      />
+      <GenericModal
+        display={showDeleteConfirmModal}
+        title={'Delete a document'}
+        message={
+          <>
+            <div>You have chosen to delete this document. </div>
+            <br />
+            <div>
+              If the document is linked to other files or entities in PIMS it will still be
+              accessible from there, however if this the only instance then the file will be removed
+              from the document store completely.
+            </div>
+            <br />
+            <strong>Do you wish to continue deleting this document?</strong>
+          </>
+        }
+        handleOk={onDeleteConfirm}
+        handleCancel={() => setShowDeleteConfirmModal(false)}
+        okButtonText="Continue"
+        cancelButtonText="Cancel"
+        show
```

Comment:
question - do we need to pass this **show** property when the modal display is controlled by he **display** props?

Reply:
I dont think so.